api/core/routes.py

```diff
@@ -495,10 +601,11 @@ def propagateSatellite(tleLine1, tleLine2, lat, lon, elevation, jd, dtsec=1):
     satellite = EarthSatellite(tleLine1,tleLine2,ts = ts)
 
     #Get current position and find topocentric ra and dec
-    currPos = wgs84.latlon(lat, lon, elevation)
+    currPos = wgs84.latlon(location.lat.value, location.lon.value, location.height.value)
     # Set time to satellite epoch if input jd is 0, otherwise time is inputted jd
-    if jd == 0: t = ts.ut1_jd(satellite.model.jdsatepoch)
-    else: t = ts.ut1_jd(jd)
+    # Use ts.ut1_jd instead of ts.from_astropy because from_astropy uses astropy.Time.TT.jd instead of UT1
+    if jd.jd == 0: t = ts.ut1_jd(satellite.model.jdsatepoch)
+    else: t = ts.ut1_jd(jd.jd)
```

Comment:
I get nervous when I see single character variables, especially when they are times. What is this time exactly, and can you please give it a slightly more descriptive variable name?